Form/Type/BooleanType.php

```diff
@@ -81,7 +81,9 @@ public function configureOptions(OptionsResolver $resolver)
      */
     public function getParent()
     {
-        return 'choice';
+        return class_exists('Symfony\Component\Form\Extension\Core\Type\ChoiceType') ?
```

Comment:
When I used the test you provided I was getting the date type instead of fully qualified name.